content/docs/user-guide/data-pipelines/index.md

```diff
@@ -0,0 +1,14 @@
+# Data Pipelines
```

Comment:
I'm not sure that the Data Pipelines is the right term

Reply:
There is not connection to models, sounds like ETL, do we have any alternatives? call it just Pipelines?

Reply:
Yeah just Pipelines, DVC Pipelines, or ML Pipelines (https://github.com/iterative/dvc.org/pull/3414#pullrequestreview-966781826)